### PR TITLE
Merge sigclient repo into siglens

### LINKED ISSUE #269: [STORY] Move sigclient repo into this repo
# Description
We have a test tool that sits in https://github.com/sigscalr/sigscalr-client, move all of the files from this repo into this repo at location `tools/sigclient`

# Acceptance Criteria
1. The tool files are moved in its own dir
2. the client's readme files contains some cmds to run the load, make sure those cmds continue to work

### PR DESCRIPTION
**Pull request by venkateshamatam**
# Description
Merging sigclient repo into the siglens repo
Fixes #269

# Testing
Merged #286 into my fork branch and removed `tools/sigclient/.gitignore` and `tools/sigclient/.github/workflows` and formatted the go code

# Checklist:
Before marking your pull request as ready for review, complete the following.

- [x] I have self-reviewed this PR.
- [x] I have removed all print-debugging and commented-out code that should not be merged.
- [x] I have added sufficient comments in my code, particularly in hard-to-understand areas.
- [x] I have formatted the code, if applicable. For Go, I have run `goimports -w .`.

### CODE REVIEW

File: tools/sigclient/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM golang:1.18-alpine3.17
```
Comment (nkunal) [inline tools/sigclient/Dockerfile:1]: we should update the alpine version here, keep it same as siglens version

File: tools/sigclient/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (nkunal) [inline tools/sigclient/LICENSE.md:1]: remove this file from here

**Review by nkunal — COMMENTED** [review]
loosk pretty good

**Review by AndrewHess — COMMENTED** [review]
Looks good to me

File: tools/sigclient/Dockerfile (hunk 1 above)
Comment (venkateshamatam) [reply tools/sigclient/Dockerfile:1]: done. updated it to match siglens repo `FROM golang:1.21-alpine3.18`

File: tools/sigclient/LICENSE.md (hunk 2 above)
Comment (venkateshamatam) [reply tools/sigclient/LICENSE.md:1]: removed `tools/sigclient/LICENSE.md`

**Review by AndrewHess — APPROVED** [review]

**Pull request merged**